animation_nodes/data_structures/midi/midi.py

```diff
@@ -0,0 +1,30 @@
+
+class MIDINote_Class:
+
+    # Channel initializations
+    def __init__(self, channel, number, time_on, time_off, velocity):
+        # Parameters
+        self.channel = channel      # channel
+        self.number = number        # number of note
+        self.time_on = time_on      # note on time
+        self.time_off = time_off    # note off time
+        self.velocity = velocity    # note on velocity
```

Comment:
Use `camelCase` for variable names. No need for comments. `number` -> `noteNumber`